src/OpenTelemetry.Exporter.OneCollector/OpenTelemetry.Exporter.OneCollector.csproj

```diff
@@ -1,10 +1,10 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <Description>An OpenTelemetry .NET exporter that sends telemetry to Microsoft OneCollector</Description>
-    <TargetFrameworks>net8.0;net6.0;netstandard2.1;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net8.0;netstandard2.1;netstandard2.0</TargetFrameworks>
```

Comment:
@joegoldman2 Any reason you didn't use `$(SupportedNetTargetsWithoutNet6)` here?

Reply:
Because no variable was used before and I think we don't want to align TFMs for all packages in this repo, right (except for example and test projects)? Otherwise, when we will add or remove a TFM, all packages will be impacted. I'm fine with using `$(SupportedNetTargetsWithoutNet6)` here otherwise.